File: sesion-5/insecure-snippets-js-csharp/csharp/2-HardcodedSecrets.cs
// ⚠️ Secretos hardcodeados en código C#
using System;

class Program
{
    static void Main()
    {
        string apiKey = "1234567890";
    // Uso inseguro: la API Key se expone directamente en una petición HTTP
    string url = $"https://api.ejemplo.com/data?apikey={apiKey}";
    Console.WriteLine($"Realizando petición a: {url}");
    // También se imprime la API Key en logs
    Console.WriteLine($"API Key: {apiKey}");
    }
}

File: sesion-5/insecure-snippets-js-csharp/js/2-hardcoded-secrets.js
// ⚠️ Secretos hardcodeados: API keys en código fuente
// Descripción: Este snippet contiene una API key hardcodeada en el código fuente, lo que puede ser explotado si el código es accesible públicamente.
const apiKey = "12345-ABCDE";  
//general un ejemplo de uso de API
fetch(`https://api.example.com/data?api_key=${apiKey}`)
  .then(response => response.json())
  .then(data => console.log(data)); 